examples/tls-with-aws-pca-issuer/main.tf

```diff
@@ -114,7 +114,9 @@ module "eks_blueprints_addons" {
   enable_cert_manager         = true
   enable_aws_privateca_issuer = true
   aws_privateca_issuer = {
-    acmca_arn = aws_acmpca_certificate_authority.this.arn
+    acmca_arn        = aws_acmpca_certificate_authority.this.arn
+    namespace        = "aws-privateca-issuer"
```

Comment:
we do we add this here? Is it this line and the next not set in the module?

Reply:
We don't need to add here. By default we are creating the PCA in the `kube-system` Namespace. I just added a different option because some docs for the `aws-privateca-issuer` are pointing to the namespace with the same name, even with no default value in the Helm Chart.